examples/hello/BUILD.bazel

```diff
@@ -0,0 +1,65 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
generally, this feels wayyy to long... All examples will have roughly the same build definition. Can we generalize this somehow (i.e. create something like `cdk8s_example(...)` command?). I feel this file should be a single line with all defaults.

Reply:
Yes we can, I'll create a macro for this in the next commit. I had this inspiration last night too, but you beat me to it 😉